static/app/views/issueList/searchBar.tsx

```diff
@@ -170,3 +170,7 @@ const RecommendedWrapper = styled('div')`
     margin: 0;
   }
 `;
+
+const StyledIconStar = styled(IconStar)`
```

Comment:
Is there a better way? Maybe a line height adjustment?

Feels like such a hack